conf/machine/pinephone.conf

```diff
@@ -45,6 +45,8 @@ IMAGE_BOOT_FILES = " \
     boot.scr \
 "
 
+do_image_wic[depends] += "initramfs-uboot-image:do_image_complete"
+
```

Comment:
Let's also add virtual/bootloader:do_deploy , what do you think?

Reply:
There is a dependency on u-boot already:
"luneui-example-image.do_image_wic" -> "u-boot.do_deploy"